config/environments/production.rb

```diff
@@ -39,7 +41,7 @@
 
   # Log to STDOUT by default
   config.logger = ActiveSupport::Logger.new($stdout)
-    .tap { |logger| logger.formatter = config.log_formatter }
+                                       .tap { |logger| logger.formatter = config.log_formatter }
```

Comment:
What cop led to this change? I'm not sure I can get behind this one, having a massive hanging indent doesn't seem right to me.